controllers/organization.js

```diff
@@ -1,3 +1,16 @@
-class OrganizationController {}
+const { Organization } = require('../models/');
+
+class OrganizationController {
+  async showPublicInformation() {
+    const findOrganization = await Organization.findOne();
```

Comment:
Podes devolver solo los campos buscados en sequelize(haciendolo más eficiente)
[Doc](https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#specifying-attributes-for-select-queries)